docs/root/intro/arch_overview/http/upgrades.rst

```diff
@@ -108,3 +111,14 @@ and `bazel-bin/source/exe/envoy-static --config-path  configs/terminate_connect.
 you will be running two Envoys, the first listening for TCP traffic on port 10000 and encapsulating it in an HTTP/2
 CONNECT request, and the second listening for HTTP/2 on 10001, stripping the CONNECT headers, and forwarding the
 original TCP upstream, in this case to google.com.
+
+HTTP/1.1 CONNECT can be used to have TCP client connecting to its own
+destination passing through an HTTP proxy server (e.g. corporate proxy):
+
+[HTTP client] --- raw HTTP --- [Envoy]  --- HTTP tunneled over HTTP/1.1  --- [Upstream HTTP Proxy]  --- raw HTTP  --- [HTTP Server]
```

Comment:
Let's have the client side and server side the same for both HTTP/1 and HTTP/2 (you can reverse either, but having upstream on the left farther above and client on the left immediately above seems confusing)